### DISCUSSION TITLE
Access to local network of the server via v2ray - VPN mode

### DISCUSSION THREAD

**Discussion by Mohammad-b0** [v2fly/v2ray-core#2585] [Q&A]
Hello,

Let’s assume I have installed v2ray on my server (Vless+WS+TLS) and provided config to the Android user to use on its client. The v2ray server has two network interfaces. 

A)	With public IPv4 Address like 1.2.3.4 that Android user uses as the server IP for v2ray config
B)	With private IPv4 Address like 10.0.0.1 

Now, I want to lunch a webserver on the interface B and aim to see my connected v2ray android clients would be able to see the webserver as a normal VPN does.

1.	Is it possible with v2ray?
2.	Which settings/documentation do I need to read?

**Comment by RadCod3** [comment]
Did you ever find an answer to this @MuhammadBor
